03_background_data_graph_info.Rmd

```diff
@@ -0,0 +1,30 @@
+# Which visualization should I use?
+
+A data visualization is made up of data and ...
+
+Deciding which data visualization to use ...
+
+```{r, include = FALSE}
+ottrpal::set_knitr_image_path()
+```
+
+## Learning Objectives
+
+This chapter will cover:
+
+- {LO1}
+- {LO2}
+
+## Data types
+
+## Graph types
```

Comment:
Per @carriewright11 's suggestion, I like showing a decision tree like this to help people decide on visualizations: https://www.data-to-viz.com/

Probably need to also expand to include cancer informatics-specific visualizations, including genomics/volcano plots, etc